plugins/palo_alto_pan_os/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: palo_alto_pan_os
 title: Palo Alto Firewall
 description: Manage Palo Alto Networks firewall devices
-version: 6.1.3
+version: 6.1.4
```

Comment:
This should be 6.1.3

Reply:
We leave version, because we added information about connection to description of every action. If you would like we can of course reuse previous version.